components/slack/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/slack",
-  "version": "0.4.10",
+  "version": "0.4.11",
```

Comment:
I think `crypto` needs to be added as a dependency in package.json. You can do that by running `npm install crypto` in the components/slack directory.

Reply:
I thought the same, but the package is now built-in on node.

```
npm install:
crypto@1.0.1: This package is no longer supported. It's now a built-in Node module. If you've depended on crypto, you should switch to the one that's built-in.
```